mod/dirfind.php

```diff
@@ -142,7 +142,7 @@ function dirfind_content(App $a, $prefix = "") {
 			$j->items_page = $perpage;
 			$j->page = $a->pager['page'];
 			foreach ($results AS $result) {
-				if (poco_alternate_ostatus_url($result["url"])) {
+				if (GContact::alternateOstatusUrl($result["url"])) {
```

Comment:
I believe this is supposed to be `PortableContact::alternateOStatusURL()`.

Reply:
done